test/e2e/clusterlogging_test.go

```diff
@@ -275,12 +275,12 @@ func clusterLoggingUpgradeClusterTest(t *testing.T, f *framework.Framework, ctx
 	newEnv := []v1.EnvVar{
 		{Name: "WATCH_NAMESPACE", ValueFrom: &v1.EnvVarSource{FieldRef: &v1.ObjectFieldSelector{FieldPath: "metadata.namespace"}}},
 		{Name: "OPERATOR_NAME", Value: "cluster-logging-operator"},
-		{Name: "ELASTICSEARCH_IMAGE", Value: "docker.io/openshift/origin-logging-elasticsearch5:upgraded"},
-		{Name: "FLUENTD_IMAGE", Value: "docker.io/openshift/origin-logging-fluentd:upgraded"},
-		{Name: "KIBANA_IMAGE", Value: "docker.io/openshift/origin-logging-kibana5:upgraded"},
-		{Name: "CURATOR_IMAGE", Value: "docker.io/openshift/origin-logging-curator5:upgraded"},
-		{Name: "OAUTH_PROXY_IMAGE", Value: "docker.io/openshift/oauth-proxy:latest"},
-		{Name: "RSYSLOG_IMAGE", Value: "docker.io/viaq/rsyslog:upgraded"},
+		{Name: "ELASTICSEARCH_IMAGE", Value: "quay.io/openshift/origin-logging-elasticsearch5:upgraded"},
```

Comment:
we technically don't need to make these changes here -- we expect that the images aren't actually available to properly start up.